networking_oneview/ml2/drivers/oneview/common.py

```diff
@@ -36,6 +36,41 @@
 
 oneview_exceptions = importutils.try_import('hpOneView.exceptions')
 
+opts = [
+    cfg.StrOpt('oneview_host',
+               help='IP where OneView is available'),
+    cfg.StrOpt('username',
+               help='OneView username to be used'),
+    cfg.StrOpt('password',
+               secret=True,
+               help='OneView password to be used'),
+    cfg.StrOpt('uplinkset_mappings',
+               help='UplinkSets to be used'),
+    cfg.BoolOpt('allow_insecure_connections',
+                default=False,
+                help="Option to allow insecure connection with OneView."),
+    cfg.StrOpt('tls_cacert_file',
+               help="TLS File Path"),
+    cfg.StrOpt('flat_net_mappings',
+               help='Flat Networks on Oneview that are managed by Neutron'),
+    cfg.IntOpt('ov_refresh_interval',
+               default=3600,
+               help='Interval between periodic task executions in seconds'),
+    cfg.BoolOpt('developer_mode',
+                default=False,
+                help='Only set this option as true if under developer mode'),
+    cfg.IntOpt('retries_to_lock_sh',
+               default=10,
+               help=('Maximum number of attempts when trying to lock Server ',
+                     'Hardware for connection creation')),
+    cfg.IntOpt('retries_to_lock_sp',
+               default=10,
+               help=('Maximum number of attempts when trying to lock Server ',
+                     'Hardware for connection creation'))
+]
+
+CONF = cfg.CONF
+CONF.register_opts(opts, group='oneview')
```

Comment:
this options and CONF was moved to networking_oneview/conf/oneview.py, please change there.

Reply:
actually, I think those parameters are better in the DEFAULT section.